ci-operator/config/openshift/origin/openshift-origin-main.yaml

```diff
@@ -673,6 +673,18 @@ tests:
       - observers-resource-watch
     workflow: openshift-e2e-gcp
   timeout: 5h0m0s
+- always_run: false
+  as: e2e-gcp-ovn-usernamespace
+  optional: true
+  steps:
+    cluster_profile: gcp-3
+    env:
+      FEATURE_SET: TechPreviewNoUpgrade
+      TEST_SUITE: openshift/usernamespace
+    observers:
+      enable:
+      - observers-resource-watch
+    workflow: openshift-e2e-gcp
```

Comment:
@stbenjam jobs that are specific to a feature which currently requires techpreview, but may not in the future. Do you know if we want those named with techpreview forcing a job rename in the future?

Reply:
I do think we want to know if a job is operating in techpreview mode or not because it has /other/ features turned on too.

If the feature goes default, they can create a non-TP job. We don't want to compare that job with the techpreview history, because it's different in some pretty significant ways.